src/components/CodeView/index.spec.tsx

```diff
@@ -170,9 +169,12 @@ describe(__filename, () => {
 
   it('renders plain text code when mime type is not supported', () => {
     const mimeType = 'mime/type';
-    const root = renderWithLinterProvider({ mimeType });
+    const { provider, renderContent } = simulateCommentableLine({
+      mimeType,
+    });
+    const root = renderContent();
```

Comment:
So `provider` will be what we previously had in `root` (i.e., before this patch), and `root` will be a rendered line of content? Maybe these could be named in a way that makes this more obvious?